script.js

```diff
@@ -0,0 +1,75 @@
+// Task 1
+for (let i = 10; i <= 25; i += 0.5) {
+  console.log(i);
+}
+
+//Task 2
+let askingNumber = prompt("Please enter prime number");
+while (askingNumber % 2 == 0) {
```

Comment:
But we do not need to ask the user to enter a number until he enters the prime one.